pyjob/tests/test_lsf.py

```diff
@@ -1,14 +1,17 @@
 __author__ = 'Felix Simkovic'
 
+import mock
```

Comment:
please import from `unittest.mock`

Reply:
Which version of `unittest` are you using? In python2.7 `unittest` has no module `mock`...